packages/loading/README.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Loading
+description: Loading indicators inform users that their action is being processed.
+group: Content
+---
+
+## Loading Blanket
+
+The `LoadingBlanket` component can be used indicate the loading state of a component or page.
```

Comment:
We should be more specific about where a loading blanket should be used. Such as before or after a redirection? Lets talk to the designers

Reply:
I've added some more documentation around this 👍